server/storage_index.go

```diff
@@ -446,17 +504,38 @@ func (si *LocalStorageIndex) queryMatchesToStorageIndexResults(dmi search.Docume
 				idxResult.Key = string(value)
 			case "user_id":
 				idxResult.UserID = string(value)
-			case "value":
-				idxResult.Value = string(value)
 			case "version":
 				idxResult.Version = string(value)
+			case "read":
+				read, vErr := bluge.DecodeNumericFloat64(value)
+				if err != nil {
```

Comment:
shouldn't you be checking `vErr != nil` here instead?

Reply:
Nice catch 👍